aiohttp/test_utils.py

```diff
@@ -326,16 +326,16 @@ class TestClient:
 
     :type app: aiohttp.web.Application
 
-    :param protocol: the aiohttp.web application passed to create_test_server
+    :param protocol: http or https
 
-    :type app: aiohttp.web.Application
+    :type protocol: str
 
     TestClient can also be used as a contextmanager, returning
     the instance of itself instantiated.
     """
 
     def __init__(self, app, protocol="http"):
-        self._app = app
+        self.app = app
```

Comment:
Since aiohttp encourages `app` holding all variables this is basically the test client's only way of getting access to serverside data.